### DISCUSSION TITLE
Purchased Github copilot subscription after technical preview - now cannot activate VScode plugin

### DISCUSSION THREAD

**Discussion by leancoderkavy** [community/community#31191] [Copilot Conversations]
I cannot activate Github copilot VSCode plugin after purchasing a year subscription of Github copilot after my technical preview has ended.

**Comment by melvincwng** [comment]
@kavymi Can you try logging out of your GitHub account in VSCode?

Then relogin back again to see if this error/issue is resolved.

Hope this helps.

**Comment by davecheney** [comment]
@kavymi i see no copilot subscription for your account. Do you have more than one GitHub account? Possibly you have purchased copilot under a different account.

**Comment by limjunggun** [comment]
Me too. I use vs code and webstorm and it works fine in webstorm. But vs code just spit the error message 

GitHub Copilot could not connect to server. Extension activation failed: "Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended."

I uninstalled stable version and install nightly version. and it works in vscode

**Reply by mintplo** [reply]
Same Problem Here.

Github Copilot Plugin display error message "Please login to Github and try again".

Github Copilot Billing Page displays

"Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended."

**Reply by davecheney** [reply]
@mintplo please contact support who will be able to correct the billing error on your account. https://support.github.com/contact
